nixos/modules/services/misc/geoipupdate.nix

```diff
@@ -99,9 +99,22 @@ in
       LockFile = "/run/geoipupdate/.lock";
     };
 
+    systemd.services.geoipupdate-create-db-dir = {
```

Comment:
Just an idea. Why not use `StateDirectory = "GeoIp"` in the main service and statically assign the `DatabaseDirectory` to `/var/lib/GeoIp`? That would get rid of all this hassle

Reply:
Well, it doesn't work with `DynamicUser`, but other than that it's mostly up to configurability vs simplicity. I don't know if more or less all services look for the databases in `/var/lib/GeoIP`, in which case I guess it would be fine to have it be static.